inspirehep/utils/conferences.py

```diff
@@ -53,7 +53,7 @@ def render_conferences_contributions(cnum):
 
 def conferences_in_the_same_series_from_es(seriesname):
     """Query ES for conferences in the same series."""
-    query = 'series:"{}"'.format(seriesname)
+    query = u'series:"{}"'.format(seriesname)
```

Comment:
While you are modifying this query, is there a safer way to escape `seriesname`? I assume that when it will contain a `"`, the call will blow up.